scrapper.py

```diff
@@ -44,20 +79,77 @@ def get_search_urls(self):
         pass
 
 
+class Article:
+    def __init__(self, article):
+        self.article = article
+
+
+class ArticleParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+
+    def parse(self):
+        response = requests.get(self.article_url)
+        with open(f'{ASSETS_PATH}/{self.article_id}_article.html', 'w') as file:
+            file.write(response.text)
+
+        article_bs = BeautifulSoup(response.text, 'lxml')
+
+        # with open('page_code.html', encoding='utf-8') as file:
+        #     response = file.read()
+        return
+
+    def _fill_article_with_text(self, article_bs):
+        return None
+
+
+# class PDFCrawler(HTMLCrawler):
+#     pass
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(ASSETS_PATH)
```

Comment:
will work for empty dictionaries only. read the docs: https://docs.python.org/3/library/os.html#os.rmdir